addon/errors.js

```diff
@@ -102,8 +102,8 @@ AbortError.prototype = Object.create(AjaxError.prototype);
  * @public
  * @extends AjaxError
  */
-export function ConflictError() {
-  AjaxError.call(this, null, 'The ajax operation failed due to a conflict');
+export function ConflictError(errors) {
+  AjaxError.call(this, errors, 'The ajax operation failed due to a conflict');
```

Comment:
I think this was an oversight. All the other 4** errors do this. However this is placed below Abort and Timeout error which probably made someone thing this is a purely client-side generated error.